get_deps.sh

```diff
@@ -156,26 +156,24 @@ if [[ $WITH_TFLITE != 0 ]]; then
 		LIBTF_URL_BASE=https://s3.amazonaws.com/redismodules/tensorflow
 		if [[ $OS == linux ]]; then
 			TFLITE_OS="linux"
-			# if [[ $GPU != 1 ]]; then
-			# 	TFLITE_BUILD="cpu"
-			# else
-			# 	TFLITE_BUILD="gpu"
-			# fi
+			 if [[ $GPU != 1 ]]; then
+			 	TFLITE_PLATFORM="cpu"
```

Comment:
Wouldn't `TFLITE_DEVICE` be more appropriate here?

Reply:
I'd argue it's a nit. In RedisAI we use PLATFORM to mean the backend - so this is analogous. However, I don't have a strong feeling about it. @DvirDukhan do you care?